src/logic/basic.lean

```diff
@@ -1599,4 +1605,14 @@ by by_cases h : P; simp [h]
 lemma ite_and : ite (P ∧ Q) a b = ite P (ite Q a b) b :=
 by by_cases hp : P; by_cases hq : Q; simp [hp, hq]
 
+lemma dite_dite_comm {C : Q → α} {D : ¬Q → α} {E : ¬P → ¬Q → α} (h : P → ¬Q) :
```

Comment:
`D` is an unused argument, right?

Reply:
Yes indeed! You caught it earlier than CI. I was puzzled why I had to use `@dite_dite_comm` to prove `ite_ite_comm`; now I no longer need the `@`!